### ISSUE TITLE
Stripe checkout 50%

### ISSUE THREAD

**Issue by scott-coates** [Scott-Coates-Org/group-project-team-red#16]
# Description
Stripe API must be 50% complete.

**Customer**
Checkout
<img width="315" alt="image" src="https://user-images.githubusercontent.com/1829591/174670454-16fb9a95-798d-4480-836c-1e72c8e03c16.png">

## Requirements
- [ ] Stripe checkout must be initiated (checkout does NOT have to be complete yet)

**Comment by JSdevS-coder** [comment]
Did I understood good: (checkout does NOT have to be complete yet) means just to add it not to make it functional?

**Comment by scott-coates** [comment]
@JSdevS-coder Yes, but let me give some extra information that I gave another team.

My suggestion is to make a dummy compnent called `stripe-checkout.js` and try to replicate this part of the mock 
<img width="380" alt="image" src="https://user-images.githubusercontent.com/1829591/177224320-57e68b98-4a0d-48ed-af9d-ea4f7ae6dcbc.png">

There is probably a react.js wrapper for stripe, so you may want to consider using that instead of coding the component all yourself.

For this sprint, the `stripe-checkout.js` component does not have to be fully functional, you can just hard code some fake product for now. Ideally, you would be able to put in a dummy credit card number and it would work with stripe's developer (not the live) environment.

**Comment by JSdevS-coder** [comment]
I'm trying to set up Stripe whole day and this is the error I get:

Uncaught (in promise) IntegrationError: Missing value for Stripe(): apiKey should be a string. 

I checked several times and went through all code several times but I can not find what is making it to appear
